src/components/rowItem/index.js

```diff
@@ -0,0 +1,19 @@
+import { Link } from "react-router-dom";
+import { HOMEPAGE } from "../../constants";
+import "./rowItem.css";
+
+function RowItem(props) {
```

Comment:
It's not a `RowItem` anymore it should be something like `ProductCard`.

Reply:
Done